docs/samples/browser-plugin-meetings/app.js

```diff
@@ -349,6 +349,7 @@ function createMeeting(e) {
       createMeetingDestinationElm.value = '';
       generalStartReceivingTranscription.disabled = false; // eslint-disable-line no-use-before-define
       refreshMeetings();
+      handleStreamPublishedState(meeting);
```

Comment:
I think the code here is only executed when user presses the "create meeting" button, but not when meeting objects are created as a result of pressing "sync meetings" button. Probably better to do it when we receive "meeting:added" event which I think should be fired in both cases.